kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/matchers/should.kt

```diff
@@ -78,7 +77,7 @@ fun <T> equalityMatcher(expected: T) = object : Matcher<T> {
          {
             val e = Expected(expected.print())
             val a = Actual(value.print())
-            failure(e, a).message ?: intellijFormatError(e, a)
+            failure(e, a).message ?: "expected:<${e.value.toString()}> but was:<${e.value.toString()}>"
```

Comment:
Why do it manually and not use intellij format ?

Reply:
I'm not sure if the intellij format is needed here? 

I wanted to get hide the intellij format. It was only used from `failures.kt`, apart from here. I made the `intellijFormatError` private to `failures.kt` now so no one would use it and think the values would diff (like I did) :)

Reply:
Okay.. seems like it's needed even though values are provided in the `AssertionFailedError`.. I'll revert that bit